assets/js/backend.js

```diff
@@ -116,5 +116,11 @@
 
 			e.preventDefault();
 		});
+
+		// Copy information for support.
+		$('#hp-copy-for-support-button').on('click', function(){
+			$(document).find('.hp-copy-for-support-content').select();
```

Comment:
If possible re-use the existing "click to copy" functionality we have, adding a textarea with a tooltip suggesting to click on the field is ok (for example at the bottom of the status table, once you match WordPress table CSS there it may look ok).

Also, please never use CSS classes or ids for JS handlers, we use data-component HTML attribute for this to separate styles, anchors and JS logic completely

Once you re-use the click-to-copy funcitonality (if the field is read-only it's available by default), please cleanup all the code specific to custom click-to-copy JS and CSS added here